packages/swarm-constants/properties/color/base.json

```diff
@@ -12,6 +12,7 @@
     "alert-red": { "value": "#C80000" },
     "beach-highlight": { "value": "#F7FEFF" },
     "tooltip-blue": { "value": "#3679E4" },
+    "confirm-green": { "value": "#07b203" },
```

Comment:
https://www.pivotaltracker.com/story/show/166450222 - color updated to use confirm green